composablejs/packages/fork-off/scripts/runtime-wasm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
+
+# Default binary version
+BINARY_VERSION=v1.0002
```

Comment:
True, typo. This script was intended for building the binary/wasm and moving it, but I'm not sure if it's a good idea and I'm not including it in the README. Given that we plan to include these files in each release, this can be avoided.